### PR TITLE
test: Performance test PAGOPA-706

### PR DESCRIPTION
**Pull request by AngeloCaporaso**
#### List of Changes

- Add cosmos client to setup cosmos container values (`payments-types`, `touchpoints`, `valid-bundles`);
- Config influxDB in UAT for Grafana Dashboards;
- Tuning helm values in UAT to satisfy target performance.

#### Motivation and Context

Performance test configuration, tuning and analysis.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: helm/values-dev.yaml [hunk 1]
```diff
@@ -8,14 +8,14 @@ microservice-chart:
     pullPolicy: Always
   livenessProbe:
     httpGet:
-      path: /actuator/health/liveness
+      path: /info
```
Comment (jacopocarlini) [inline helm/values-dev.yaml]: use actuator. `Info` is not ready until application started

**Review by jacopocarlini — CHANGES_REQUESTED** [review]

**Review by jacopocarlini — APPROVED** [review]
Great Job!  🚀

**Comment by jacopocarlini** [comment]
update_code

**Comment by AngeloCaporaso** [comment]
update_code

**Pull request merged**